docs/en/stack/ml/df-analytics/ml-dfa-phases.asciidoc

```diff
@@ -0,0 +1,72 @@
+[role="xpack"]
+[[ml-dfa-phases]]
+=== How a {dfanalytics-job} works
+
+This page provides high-level background information on how {dfanalytics-jobs} 
+work, what kind of phases the jobs have, and what happens during the individual 
+phases.
+
+
+[discrete]
+==== The phases of a {dfanalytics-job}
+
+During its life cycle, a {dfanalytics-job} goes through four phases regardless 
```

Comment:
I don't think the "regardless..." is necessary, or if it is I think it should be a new sentence.
```suggestion
During its life cycle, a {dfanalytics-job} goes through four phases.
```